website/docs/general/building-apisix.md

```diff
@@ -0,0 +1,258 @@
+---
+id: building-apisix
+title: Building APISIX
```

Comment:
Thanks! How about using `Building Apache APISIX from source codes`? or pick another one to improve SEO 😄